drivers/ethernet/eth_sam_gmac_priv.h

```diff
@@ -24,7 +24,9 @@
 #define GMAC_DESC_ALIGNMENT               4
 #endif
 /** Total number of queues supported by GMAC hardware module */
-#define GMAC_QUEUE_NO                     3
+#define GMAC_QUEUE_NO_REV_A               3
```

Comment:
I'd suggest to take a chance to clean up naming here. "NO" is a number of a particular queue. "NUM" is the total number of queues.